EIPS/eip-160.md

```diff
@@ -7,14 +7,20 @@ Category: Core
 Status: Final
 Created: 2016-10-20
 ```
+### Hard fork
+[Spurious Dragon](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-607.md)
+
+### Parameters
+- `FORK_BLKNUM`: 2,675,000
+- `CHAIN_ID`: 1
 
 ### Specification
 
 If `block.number >= FORK_BLKNUM`, increase the gas cost of EXP from 10 + 10 per byte in the exponent to 10 + 50 per byte in the exponent.
 
 ### Rationale
 
-Benchmarks suggest that EXP is currently underpriced by a factor of about 4-8.
+Benchmarks suggest that EXP is currently underpriced by a factor of about 4–8.
```

Comment:
We are not trying to mimic a good LaTeX document here.  Are we?

Reply:
Haha, I notice little things like this as I read it.

Reply:
I'll think when somebody complains about non-ascii characters.